src/types/types.ts

```diff
@@ -141,14 +141,6 @@ export interface ExtendedSelections extends stardust.ObjectSelections {
   removeListener: (name: string, callback: () => void) => void;
 }
 
-export interface ExtendedTheme extends stardust.Theme {
-  name: () => string;
```

Comment:
btw, I've noticed that we still keep `name()` property [at here](https://github.com/qlik-oss/nebula-table-utils/blob/6003a8a04a64655410b95131ab04714dc0f175b8/src/hooks/use-extended-theme/types.ts#L11), but stardust Theme has it already

just as a reminder for a next small refactor pr in shared lib

<img width="1311" alt="Screenshot 2023-09-21 at 12 29 46" src="https://github.com/qlik-oss/sn-pivot-table/assets/29652890/3e66b75f-0e43-4b2c-b0ec-d51b94d367cc">